### ISSUE TITLE
Run to achieve final accuracy

### ISSUE THREAD

**Issue by seelam** [uber/horovod#191]
With Horovod, have you run any models to achieve the final accuracy reported in the papers. Like 75.9% for Resnet50? What parameters have you guys used? Appreciate any pointers.

**Comment by alsrgv** [comment]
@seelam, our https://github.com/uber/horovod/blob/master/examples/keras_imagenet_resnet50.py example achieves 75.66% - 75.88% top-1 and 92.65% - 92.66% top-5 accuracy without modification on 128 GPUs across three experiments.  I believe it's possible to get better accuracy if one faithfully reproduces data augmentations and multi-crop validation used in the original ResNet-50 paper.

**Comment by seelam** [comment]
thanks @alsrgv Any other models to final accuracy? like VGG16, Inception?

**Comment by alsrgv** [comment]
@jeevan, we also trained the NMT model from TensorFlow website and a number of internal models.

**Comment by seelam** [comment]
Good to know. How you trained with any of the [HPM](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks) code?

Thanks for your answers.

**Comment by alsrgv** [comment]
There's a bug in HPM that we found in #173 regarding random seeds.  I haven't tried to run it to convergence, but I believe it should work with that bug fixed.
